docs/src/rfc/text/0000-runtime-services-support.md

```diff
@@ -0,0 +1,161 @@
+Patina uses. Runtime services must exist after boot services have been unloaded,
+and so it is not possible for the runtime services to exist as part of the
+monolithic binary, at least without very specialized handling. Runtime services
+must have a focused roadmap to align with the goals and development models of
+Patina to be clear about expected investments.
+
+## Technology Background
+
+Currently the EDKII implementation dispatches runtime modules as part of DXE phase,
+most notably RuntimeDxe which provides core services such as `SetVirtualAddressMap`
+and other modules to implement more platform specific runtime services. The original
+implementation on Patina provided an implementation of `SetVirtualAddressMap` within
+the monolithic core, but this results in some problems on Windows boot as we will
+leverage boot services memory in `SetVirtualAddressMap` which is a runtime service
+and executes after ExitBootServices. Generally, this makes assumptions about OS
+behavior which may not be sustainable. These were resolved in the following PRs.
+
+- [dxe_core: Move runtime event handling into runtime module](https://github.com/OpenDevicePartnership/patina/pull/738)
+- [patina_dxe_core: Switch to use external RuntimeDxe implementation](https://github.com/OpenDevicePartnership/patina/pull/769)
+
+The approach that has been taken in the interim is to remove the Patina
+implementation for now and to ask platforms to go back to the EDKII RuntimeDxe.
+
+## Goals
+
+1. Set short term support plan for runtime services/modules.
+2. Enumerate expansion plans for runtime services in the future.
+
+## Requirements
+
+1. Runtime services must continue to operate as expected for short term.
+2. Runtime modules must exist purely in runtime service memory.
+3. Long term goals should remove dependence on EDKII runtime code.
+
+## Unresolved Questions
+
+- Is there a suitable type of dynamic linking that may make other options suitable
+  without excessive one-off infrastructure?
+
+## Prior Art (Existing PI C Implementation)
+
+The current C implementation consists of one primary runtime module, usually
+RuntimeDxe and a variable amount of additional runtime modules. These modules are
+dispatched by DXE, but will be loaded in runtime memory. The primary runtime module
+should produce the runtime architecture protocol, defined in the PI spec section
+12.8. This protocol allows for the core to fill in critical information for the
+runtime module to implement SetVirtualAddressMap. This includes the image list of
+runtime modules, events registered for virtualizing memory, and memory map
+information that is not currently used. When `SetVirtualAddressMap` is called, the
+runtime DXE module will use the information from the protocol it published and the
+core filled to re-relocate the images, virtualizing their execution, and call all
+events registered to allow drivers/libraries to virtualize any stored pointers.
+
+## Alternatives

```

Comment:
I may have missed it, but what is the current proposed solution versus listing of alternatives?